agent/lib/internal_address_source.js

```diff
@@ -26,16 +26,9 @@ function extract_spec(def) {
     if (def.spec === undefined) {
         console.error('no spec found on %j', def);
     }
-    var o = myutils.merge(def.spec, {status:def.status});
+    var o = myutils.merge({}, def.spec, {status:def.status});
     o.name = def.metadata ? def.metadata.name : def.address;
-    if (def.metadata && def.metadata.annotations && def.metadata.annotations['enmasse.io/broker-id']) {
-        var broker_id = def.metadata.annotations['enmasse.io/broker-id'];
-        var cluster_id = def.metadata.annotations['cluster_id'];
-        if (cluster_id === undefined) {
-            cluster_id = broker_id;
-        }
-        o.allocated_to = [{clusterId: cluster_id, containerId: broker_id, state: 'Active'}];
-    }
+
```

Comment:
it looks like the enmasse.io/broker-id dealt with some backward compatibility.   Is it certain this can be safely removed/

Reply:
Yes, it is backward compatability version older than 0.26. We have already removed the compatibility in standard-controller too.